<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:&quot;#222&quot;; color: &quot;#eee&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>881</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nueva Venta:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menu_btn">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ccd1d1; color: #222; border-radius: 8px;
padding: 4px
</string>
       </property>
       <property name="text">
        <string>Men√∫</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="limpiar_venta">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>34</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ccd1d1; color: #222; border-radius: 8px;
padding: 4px;</string>
       </property>
       <property name="text">
        <string>Limpiar venta</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="agg_producto">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>34</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ccd1d1; color: #222; border-radius: 8px;
padding: 4px;</string>
       </property>
       <property name="text">
        <string>Agregar Producto</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="eliminar_producto">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>34</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ccd1d1; color: #222; border-radius: 8px;
padding: 4px;</string>
       </property>
       <property name="text">
        <string>Eliminar producto</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="realizar_venta">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>34</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#ccd1d1; color: #222; border-radius: 8px;
padding: 4px;</string>
       </property>
       <property name="text">
        <string>Realizar venta</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="ventas_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>880</width>
      <height>440</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>880</width>
      <height>440</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>880</width>
      <height>440</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#000; background: #fff;</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>170</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="total_venta_lbl">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>261</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Total: $0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="warning_lbl">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>640</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: yellow;</string>
    </property>
    <property name="text">
     <string>Ninguna fila seleccionada</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="warning2_lbl">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: yellow;</string>
    </property>
    <property name="text">
     <string>No hay mas productos en el inventario</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
